package.json

```diff
@@ -1,20 +1,20 @@
 {
   "name": "magic-script-typings",
-  "version": "1.6.2",
+  "version": "1.6.3",
   "description": "Typescript definitions for MagicScript",
   "files": [
+    "egl",
     "gl",
     "jpeg",
     "lumin",
     "mxs",
     "png",
-    "uv",
-    "index.d.ts",
-    "index.js"
+    "ssl",
+    "uv"
   ],
   "scripts": {
     "build": "node index.js",
-    "typedoc": "typedoc ./lumin",
+    "typedoc": "typedoc ./png ./jpeg ./uv ./gl ./egl ./ssl ./lumin",
```

Comment:
we're missing `mxs`

Reply:
please use the same list and order as above in `files`

Reply:
Done. Thanks.